### ISSUE TITLE
Facebook scroll lag

### ISSUE THREAD

**Issue by imdylan0831** [Alex313031/Thorium-Win-AVX2#99]
**System Details**
 - OS: Windows 11
 - Thorium Version 115.0.5790.172

**Problem**
When scrolling in news feed and comments in Facebook, browser suddenly lags and makes the buffering mouse cursor thing that is evident in generally any application that hangs on windows. I don't know where it's coming from but I'm sure that other websites don't do this. Every website I go loads fast and scrolls fast on the thorium browser, except facebook with the news feed and all... Kind of starting to become a pet peeve lol

**Comment by gz83** [comment]
Does other browsers have the same issue? Also, does this issue also occur in incognito mode?

@imdylan0831

**Comment by imdylan0831** [comment]
I don't experience it in other browsers and I don't access facebook in incognito often to notice. (but so far nothing in incognito lags for me anyway, I just experience this wierd buffering thing with facebook). Also it somehow only happens on the facebook.com domain, i.e news feed, or when going to its subdomain facebook.com/messages. It gets wierder because the random buffering never happens for messenger.com which is the standalone site of facebook for its facebook messenger.

**Comment by gz83** [comment]
Does your browser have extensions installed? If there are extensions installed, can you troubleshoot this by turning off and on the extensions you have installed one by one? Sometimes extensions can cause browsers or web pages to behave strangely.

@imdylan0831

**Comment by imdylan0831** [comment]
Oh my thorium browser not have too many extension. Its just Ublock Origin and Internet Download Manager... Actually come to think of it, it may have to do with Dark Reader extension... I made this post 2 days ago and maybe forgot that I removed Dark reader from thorium. I just tried facebook again and it doesn't do the wierd buffering thing anymore. So maybe Dark reader for thorium not really good, (now I use chrome://flags force dark mode feature and it works very lightly).

But yeah Dark reader doesn't do random hangs with Firefox for example, since I use both interchangeably.
